core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
Re compilation of `override val`: https://groups.google.com/d/topic/scala-user/7b7mtNkd194/discussion

I've shot myself in the foot defining classes (different from traits) with abstract vals before, and then trying to use the value too early. I pretty much stick with abstract defs now.